packages/fuchsia_ctl/lib/src/ssh_key_manager.dart

```diff
@@ -32,16 +34,36 @@ class SystemSshKeyManager implements SshKeyManager {
   const SystemSshKeyManager({
     this.processManager = const LocalProcessManager(),
     this.fs = const LocalFileSystem(),
+    this.pkeyPubPath,
   })  : assert(processManager != null),
         assert(fs != null);
 
+  /// Creates a static provider that returns a SystemSshKeyManager.
+  static SshKeyManager defaultProvider(
+      {ProcessManager processManager, FileSystem fs, String pubKeyPath}) {
+    return SystemSshKeyManager(
+        processManager: processManager ?? const LocalProcessManager(),
+        fs: fs,
+        pkeyPubPath: pubKeyPath);
```

Comment:
comma after `pubKeyPath`.  Also, make the parameter match - `publicKeyPath`.

Reply:
done